lib/kuzzle/kuzzle.js

```diff
@@ -28,7 +28,7 @@ const stringify = require('json-stable-stringify');
 const Koncorde = require('koncorde');
 
 const config = require('../config');
-const EntryPoints = require('../core/network');
+const EntryPoint = require('../core/network/entryPoint');
```

Comment:
(note / no action required) This change was unnecessary: soon this will be loosely coupled ;-)